apps/wing/src/commands/test.ts

```diff
@@ -62,36 +62,36 @@ function printResults(
   const durationInSeconds = duration / 1000;
   const totalSum = failing.length + passing.length;
   console.log(" "); // for getting a new line- \n does't seem to work :(
-  console.log(`
-${totalSum > 1
-      ? `Tests Results:
-${passing.map((testName) => `    ${chalk.green("✓")} ${testName}`).join("\n")}
-${failing.map(({ testName }) => `    ${chalk.red("×")} ${testName}`).join("\n")}
-${" "}
-`
-      : ""
-    }
-${failing.length && totalSum > 1
-      ? `
-${" "}
-Errors:` +
-      failing
-        .map(
-          ({ testName, error }) => `
-
-At ${testName}\n ${chalk.red(error.message)}
-        `
-        )
-        .join("\n\n")
-      : ""
-    }
-
-${chalk.dim("Tests")}${failing.length ? chalk.red(` ${failing.length} failed`) : ""}${failing.length && passing.length ? chalk.dim(" |") : ""
-    }${passing.length ? chalk.green(` ${passing.length} passed`) : ""} ${chalk.dim(`(${totalSum})`)} 
-${chalk.dim("Duration")} ${Math.floor(durationInSeconds / 60)}m${(durationInSeconds % 60).toFixed(
-      2
-    )}s
-`);
+  const areErrors = failing.length > 0 && totalSum > 1;
+  const showTitle = totalSum > 1;
+
+  const results = [
+    showTitle && `Tests Results:`,
+    // prints a list of the tests names with an icon
+    ...(showTitle ? passing.map((testName) => `    ${chalk.green("✓")} ${testName}`) : []),
+    ...(showTitle ? failing.map(({ testName }) => `    ${chalk.red("×")} ${testName}`) : []),
+    // prints error messages form failed tests
+    areErrors && " ",
+    areErrors && `Errors:`,
+    ...(areErrors
+      ? failing.map(({ testName, error }) => `At ${testName}\n ${chalk.red(error.message)}`)
+      : []),
+    " ",
+    // prints a summary of how many tests passed and failed
+    `${chalk.dim("Tests")}${failing.length ? chalk.red(` ${failing.length} failed`) : ""}${
+      failing.length && passing.length ? chalk.dim(" |") : ""
+    }${passing.length ? chalk.green(` ${passing.length} passed`) : ""} ${chalk.dim(
+      `(${totalSum})`
```

Comment:
Looks like `failing.length` and `passing.length` are always 1 (look at the snapshots).

Reply:
yes, if you run only one test at a time, if you run a few (using `*` in the path) it will show you the number of test files per folder, for examples:
<img width="792" alt="image" src="https://github.com/winglang/wing/assets/39455181/570f6b9c-63ef-41b1-9503-5bcd5258180f">

Reply:
Can you see if it also closes #2946?

Reply:
no, since I count files there, I think I can also count tests instead, will it be better?

Reply:
done :)
<img width="838" alt="image" src="https://github.com/winglang/wing/assets/39455181/073917fd-ca72-4008-a599-9814745a0366"> (temporary added one failing test to one of the files, to see the printed results)
<img width="826" alt="image" src="https://github.com/winglang/wing/assets/39455181/660a4029-fe97-4160-a4b8-af510c45ca04">